docs/reference/transform/limitations.asciidoc

```diff
@@ -33,6 +33,14 @@ upgrade from 7.2 to a newer version, and {transforms} have been created in 7.2,
 the {transforms} UI (earler {dataframe} UI) will not work. Please wait until all 
 nodes have been upgraded to the newer version before using the {transforms} UI.
 
+[float]
+[[transform-rolling-upgrade-limitation]]
+==== {transforms-cap} Transforms suspended during a rolling upgrade from 7.2 and 7.3
+
+If your cluster contains mixed version nodes, for example during a rolling
+upgrade from 7.2, 7.3 to a newer version, transforms are suspended for the time of
```

Comment:
Is "suspended" the right word here?  I thought that we would not assign transform tasks in a mixed version cluster that includes pre-7.4 nodes, but we will not actually _suspend_ transforms that are already running before the upgrade starts.

If so the the issue title could be "Transforms reassignment suspended during a rolling upgrade from 7.2 and 7.3", and "transforms are suspended for the time of the upgrade" could be "transforms whose nodes are stopped will not be reassigned until the upgrade is complete".